modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -873,7 +873,7 @@ AUI.add(
 						);
 					}
 				}
-				else if (dataType === 'double' || dataType === 'integer') {
+				else if (dataType === 'date' || dataType === 'double' || dataType === 'integer') {
```

Comment:
Group this expressions with parentheses, and you don't need to use `===`, just `==`unless it is to compare data types too.

Reply:
ok